torchgen/model.py

```diff
@@ -1183,6 +1187,20 @@ def strip_ret_annotation(r: Return) -> Return:
         if strip_view_copy_name and base_name.endswith("_copy"):
             base_name = base_name.replace("_copy", "")
 
+        if convert_mutable_inputs_to_returns:
+            # find mutable inputs that are not originally returned, and conver them to returns
+            returns_from_mutable_inputs = tuple(
+                Return(name=None, type=a.type, annotation=None)
```

Comment:
Sure you want no name here? It sure seems like it would be useful for the test case you gave above

Reply:
oh yeah - although I think I actually want no names here, that way I can properly pair up the functional + mutable ops and emit a proper error if someone didn't name the outputs properly. (If I add names here, then we would just silently not pair up the two ops instead of emitting an error)